spec/system/candidate_interface/references/candidate_submits_a_duplicate_application_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.feature 'Submitting an application' do
+  include CandidateHelper
+
+  scenario 'Candidate submits complete application' do
+    given_i_am_signed_in
+    and_i_have_completed_my_first_application
+
+    when_i_submit_the_application
+    then_i_can_see_my_application_has_been_successfully_submitted
+
+    when_i_am_signed_in_using_a_new_account
+    and_i_complete_a_second_duplicate_application
+    and_the_duplicate_matching_service_runs
+    then_i_can_see_a_warning_message
+  end
+
+  def given_i_am_signed_in
+    create_and_sign_in_candidate
+  end
+
+  def when_i_am_signed_in_using_a_new_account
+    @current_candidate = create(:candidate)
+    create_and_sign_in_candidate
+  end
+
+  def and_i_have_completed_my_first_application
+    candidate_completes_application_form
+  end
+
+  def and_i_complete_a_second_duplicate_application
+    candidate_completes_application_form
+  end
+
+  def and_i_submit_the_application
+    visit candidate_interface_application_form_path
+    click_link 'Check and submit your application'
+    click_link t('continue')
+  end
+  alias_method :when_i_submit_the_application, :and_i_submit_the_application
+
+  def then_i_can_see_my_application_has_been_successfully_submitted
```

Comment:
Do we need to complete the questionnaire?

Reply:
I just wanted to submit the application and I don't think that happens until the questionnaire is filled in.